realm/build.gradle

```diff
@@ -26,6 +29,8 @@ android {
 }
 
 dependencies {
+    compile 'com.android.support:multidex:1.0.0'
+    compile 'it.unimi.dsi:fastutil:7.0.7'
```

Comment:
Is adding a dependency to an external library a good idea? @cmelchior @nhachicha do you have experiences with it?